package.json

```diff
@@ -84,5 +88,6 @@
         "dist/**/*",
         "LICENSE",
         "README.md"
-    ]
+    ],
+    "packageManager": "yarn@3.4.1+sha512.a333bf9c2557d71f5ba87dd1446cde24bb9c35301ec29dd2baba577b3f9d1ab61348b66a1324de9088c40a996c5b9203794da8bf14cab9fae6a6eccc00cb8078"
```

Comment:
Can you remove this, it makes the build crash

You're right that we're using yarn v1. We should probably switch to npm now, but that's a task for another day

Reply:
Oh man, I must have checked for this three times. Kept having to revert changes to yarn config and such, even in yarn v1 mode.